### PR TITLE
doc: Invite users to donate to Gridcoin Foundation

### PR DESCRIPTION
**Pull request by smoe**
Hello, as promised on Discord, here the PR that calls for donations. Please kindly adjust the language to whatever a native-speaking mind might want to hear. I would really like to also add a URL so aspiring donors can learn about what is going on. Instead of this rather blunt statement on the front page of the help system, there could be a separate section "donations" but one needs to start somewhere and respective implementations seem trivial.

This is what it looks like
```
$ ./src/gridcoinresearchd help
help [command/category]
Returns help on a specific command or category you request

help command --> Returns help for specified command; ex. help backupwallet

Categories:
wallet --------> Returns help for blockchain related commands
mining --------> Returns help for staking/cpid/beacon related commands
developer -----> Returns help for developer commands
network -------> Returns help for network related commands
voting --------> Returns help for voting related commands

The Gridcoin Foundation invites to donate to address
bc3NA8e8E3EoTL1qhRmeprbjWcmuoZ26A2
to foster the Gridcoin development.
```

### CODE REVIEW

**Comment by Moohan** [comment]
Suggested wording, something like 
> You can support the development of Gridcoin by gifting a small amount to the Gridcoin Foundation at this address: bc3NA8e8E3EoTL1qhRmeprbjWcmuoZ26A2

Some pointers on asking for donations here: [https://www.soapboxengage.com/blog/1806-5-ways-to-boost-your-wording-when-asking-for-donations](https://www.soapboxengage.com/blog/1806-5-ways-to-boost-your-wording-when-asking-for-donations)

**Comment by dblanch256** [comment]
I like it!  Making a small donation right now.

David C Blanchard
MIT '76
609.238.3895

*"If I have seen a little farther than others, it is because I have stood
on the shoulders of giants"*   -- Isaac Newton


*"If I have not seen as far as others, it is because giants were standing
on my shoulders."*  -- Hal Abelson


*"In the field of Computer Science, we stand on each others' feet."*  --
Richard Hamming

File: src/rpc/server.cpp [hunk 1]
```diff
@@ -217,7 +217,11 @@ UniValue help(const UniValue& params, bool fHelp)
             "mining --------> Returns help for staking/cpid/beacon related commands\n"
             "developer -----> Returns help for developer commands\n"
             "network -------> Returns help for network related commands\n"
-            "voting --------> Returns help for voting related commands\n";
+            "voting --------> Returns help for voting related commands\n"
+	    "\n"
+	    "The Gridcoin Foundation invites to donate to address\n"
+	    "bc3NA8e8E3EoTL1qhRmeprbjWcmuoZ26A2\n"
```
Comment (jamescowens) [inline src/rpc/server.cpp]: Change the wording to...

You can support the development of Gridcoin by donating GRC to the Gridcoin Foundation at this address:

**Review by jamescowens — CHANGES_REQUESTED** [review]

File: src/rpc/server.cpp (hunk 1 above)
Comment (jamescowens) [reply src/rpc/server.cpp]: Did you change it and re-push?

Comment (smoe) [reply src/rpc/server.cpp]: It just arrived. I put the line break just before "Gridcoin Foundation" - did not want to separate these words. Googled the hash - looks fine to me.

**Review by jamescowens — APPROVED** [review]
Perfect! :)

**Comment by smoe** [comment]
Much in analogy to what the Discord Gridcoin wrapper offers, I envision a menu in the official Gridcoin wallet that instructs how to donate to various scientific projects. That list of projects does not necessarily need to be synced with the whitelist. My motivation for such then official wallets for BOINC projects is to have an objective function to assess how much the  individual projects are supported by the community. This would be of interest for the project maintainers (who otherwise only have the #hosts/#hours to learn if they are doing everything right) and also for the Gridcoin as a future paramter to tap into to decide how to transform magnitude to GRC.

Such a donation-page could start statically within the GRC wallet, but preferably this would be scraped from a trusted website somewhere?

**Pull request merged**